defaults/main.yml

```diff
@@ -14,6 +14,7 @@ postgresql_default_auth_method: "trust"
 # The user/group that will run postgresql process or service
 postgresql_service_user: "{{ postgresql_admin_user }}"
 postgresql_service_group: "{{ postgresql_admin_user }}"
+postgresql_service_enabled: false
```

Comment:
The difference is probably due to how the package is installed on the different OS flavors.

I'd say enable it by default and we'll bump the major version component when we merge :)